.github/actions/maven-publish/action.yml

```
@@ -33,7 +35,7 @@ runs:
 
     - name: Build Project
       shell: bash
-      run: ./gradlew clean assemble -PisSnapshot=false
+      run: ./gradlew clean assemble -PisSnapshot=false -Pversion="${{ inputs.version }}
```

Is there a missing `"` at the end here?
```suggestion
      run: ./gradlew clean assemble -PisSnapshot=false -Pversion="${{ inputs.version }}"
```